report/main.qmd

```diff
@@ -0,0 +1,139 @@
+| Zero-calorie beverages sold | Continuous  |
+|    Other beverages sold     | Continuous  |
+
+: **Description of variables** {#tbl-vars tbl-colwidths="\[35,25\]"}
+
+In addition to the periods that were not recorded at the beginning of the study in sites B and C, there are nine missing values for sales of zero-calorie and sugary beverages. The missing observations correspond to the last week of site B and two days of site C. Aside from the missing information at the beginning and end of the study, the other missing values are unrelated to any specific factor. Furthermore, the sales data for other beverages and the total amount have several missing values, but they do not affect this analysis.
+
+# Exploratory Analysis
+
+Given that the data consists of a time series of sales across three sites, it was necessary to carry out a time-based analysis. In that sense, @fig-1 helps visualize the beverages sold and the shadows behind the lines display the distinct intervention periods by site. Additionally, @sec-appeda summarizes the effect from past observations in the new data points and the decomposition of sugary and zero-calorie sales series in the change by the mean level (trend), the periodicity of the data (seasonal variation) and factors that do not show a pattern (random variation).
+
+![**Sale of sugary and zero-calorie drinks by intervention**](../figs/eda-1_time-serie.png){#fig-1}
+
+In particular, @fig-1 shows some important characteristics of the dataset. Firstly, the measurements for each site began at different times. Secondly, the third site experienced a significant increase in sales during most of the first intervention, but afterwards, sales remained at a lower and more stable level. Thirdly, the order of the three calorie messaging interventions was different for each site. Lastly, it is evident from the data that there is a weekly seasonal effect.
+
+# Formal analysis
+
+To evaluate the research questions, it is suggested to utilize a preparation step along with two statistical methods. In the following subsections, the necessary data adjustments, a detailed explanation of each recommended model and the suggested procedure to conduct and analyze the results are presented.
+
+## Pre-analysis
+
+Before conducting the statistical analysis, it is important to address some elements that were identified during the Exploratory Analysis that could impact the results. These include the weekly effect, different lengths of the pre-intervention observations and unnecessary information. To focus on capturing the intervention effects instead of the number of days effects, one can eliminate the weekly impact by either using a percentage of sales or decomposing the time series, as suggested by @ChatfieldXing:2019. While the percentage approach may be easier, it is recommended to use the decomposition method as the variable containing the total items sold may have inconsistencies that suggest unreliable information.[^2]
```

Comment:
It is good to identify the trend or seasonality in some time-series in panel data, but since it is slightly misleading since the methods you suggest will not model trend, seasonality and random variation separately. A better idea is to identify which covariates can help to explain those variations in data (like day of week), instead of emphasizing the idea of decomposing time-series too much.